pkg/rpc/server/server_test.go

```diff
@@ -241,16 +235,16 @@ func TestCacheServer_PutArtifact(t *testing.T) {
 					},
 				},
 			},
-			putImage: cache.ArtifactCachePutArtifactExpectation{
-				Args: cache.ArtifactCachePutArtifactArgs{
-					ArtifactID: "sha256:e7d92cdc71feacf90708cb59182d0df1b911f8ae022d29e8e95d75ca6a99776a",
-					ArtifactInfo: ftypes.ArtifactInfo{
-						SchemaVersion: 1,
-						Created:       time.Date(2020, 1, 2, 3, 4, 5, 6, time.UTC),
-					},
-				},
-				Returns: cache.ArtifactCachePutArtifactReturns{
-					Err: xerrors.New("error"),
+			setUpCache: func(t *testing.T) cache.Cache {
+				return cachetest.NewErrorCache(cachetest.ErrorCacheOptions{
+					PutArtifact: true,
+				})
+			},
+			wantArtifact: cachetest.WantArtifact{
+				ID: "sha256:e7d92cdc71feacf90708cb59182d0df1b911f8ae022d29e8e95d75ca6a99776a",
+				ArtifactInfo: ftypes.ArtifactInfo{
+					SchemaVersion: 1,
+					Created:       time.Date(2020, 1, 2, 3, 4, 5, 6, time.UTC),
```

Comment:
We don't check this because we returned an error.

Reply:
Deleted https://github.com/aquasecurity/trivy/pull/8403/commits/fcb112f0ccb662f1630dcf6a632ef7d908e4240f